packages/web-components/src/components/filter-panel/__stories__/filter-panel.stories.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit-element';
+import { text, select } from '@storybook/addon-knobs';
+import '../filter-panel';
```

Comment:
Just curious if we should just pull the `index.ts` instead to squash the component imports to just one line?